portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,12 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1 class="name">I AM MOSHOOD SAKA.</h1> 
+      <img class="soon" src = "/images/soon.png"/>
```

Comment:
"soon" is a fairly generic tag. Consider making this more specific about what it is being used for on the page similar to how you see ids such as "greeting-container" below.

Reply:
changed the naming to "web-in-progress"